Tests/kaas/k8s-default-storage-class/k8s-default-storage-class-check.py

```diff
@@ -25,8 +25,11 @@
 import time
 import json
 import logging
+import os
+import inspect
```

Comment:
it hasn't been sorted before, but please make it sorted

- builtin Python modules, sorted by alphabet
- blank line
- modules from dependencies, sorted by alphabet
- blank line
- modules from the same source tree, sorted by alphabet

Reply:
done (isort)